### ISSUE TITLE
Enable aria-live attribute on form errors to be configurable

### ISSUE THREAD

**Issue by abigailjxn** [verbb/formie#2505]
### What are you trying to do?

Related to [Issue 1650](https://github.com/verbb/formie/issues/1650)

For some browser and screen reader combinations, the error messages on form fields are not announced right after the error appears. I would like to adjust the `aria-live` attribute to be `assertive`, though there currently is not a way to do this. 

### What's your proposed solution?

Again, related to what was brought up in [Issue 1650](https://github.com/verbb/formie/issues/1650).

It would be great if the `aria-live` attribute (set [here](https://github.com/verbb/formie/blob/ce719b2e308e4b87ea621ae4a6cd4efbbccb0a3e/src/web/assets/frontend/src/js/validator/validator.js#L192)) could be adjustable either on the individual field level or the form level. Either something in the control panel or even just a property in the config would work. 

Perhaps it could be set within the validator rules like
```
export default {
    rule,
    message,
    ariaLiveStatus
};
```
and then read in the `showError` function in `validator.js`.

Any thoughts on if this would be reasonable?

**Comment by engram-design** [comment]
More than happy to make this configurable in some fashion. I'll have a think about exactly where this sits, but I think we'll provide options for it globally, per-form and per-field.

**Comment by engram-design** [comment]
Added in [4.0.0-beta.7](https://github.com/verbb/formie/releases/tag/4.0.0-beta.7).
